src/test/store/js-tracer.test.js

```diff
@@ -160,6 +162,99 @@ describe('selectors/getJsTracerTiming', function() {
       ]);
     });
   });
+
+  describe('match function names', function() {
+    it('can match function names from the sampled data', function() {
```

Comment:
I remember you were saying that it's better for the `describe` + `it` blocks to form a sentence. Can we do the same here?

Reply:
Touché!